### DISCUSSION TITLE
How to preview MirrorMaker2 lag on source cluster

### DISCUSSION THREAD

**Discussion by EmiliaNowakowska** [strimzi/strimzi-kafka-operator#6227] [Q&A] [answered]
Hello, 
I have Strimzi MirrorMaker2 configuration. 
Is there a way to monitor MirrorMaker2 lag on source cluster? 
Unfortunately as I checked there is no consumer group created for MirrorMaker2 on source cluster. 

Is there a way to create MirrorMaker2 consumer group on source cluster to have the possibility to monitor lag?
Do you have some other suggestion how to monitor the MirrorMaker2 replication?

**Comment by scholzj** [comment, marked as answer]
Strimzi does not offer anything special with regards to this. It just runs Mirror Maker 2 as provided by Apache Kafka. Isn't some lag information provided as part of the Mirror Maker 2 metrics?

**Reply by EmiliaNowakowska** [reply]
Unfortunately there is no lag information provided in the MM2 metrics.

Thank you for your quick answer

**Reply by scholzj** [reply]
Wait, I search the PRs and found this #5348 which was added in 0.25. That should help at least a little bit, or? (I know it is not on the source cluster, but at least something)

**Reply by EmiliaNowakowska** [reply]
Yeah, it's a useful hint how to handle it. Thanks!
